server/migrations/usersMigration.js

```diff
@@ -0,0 +1,36 @@
+import { MongoClient, ObjectId } from 'mongodb'
+import config from '../configs/config.js'
+import getMongoURI from '../utils/mongoUtil.js'
+import { calculatePreferences } from './helpers.js'
+
+async function migrateUsers() {
+  try {
+    const mongoURI = getMongoURI({ settings: config.database.mongo })
+    const mongo = new MongoClient(mongoURI, config.database.mongo.server)
+    const appDatabase = mongo.db()
+    await appDatabase
+      .collection('users')
+      .find({})
+      .forEach((user) => {
+        const userPreferences = calculatePreferences(user)
+
+        appDatabase.collection('users').updateOne(
+          { _id: ObjectId(user._id) },
+          {
+            $set: {
+              preferences: userPreferences,
```

Comment:
should we keep legacy preferences for now to make sure the migration works?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241